src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb

```diff
@@ -94,9 +94,7 @@
     "idaes.cfg.ipopt.options.nlp_scaling_method = \"user-scaling\"\n",
```

Comment:
On a line I can't comment (the very top Markdown cell), there is a typo "includeds". In the next Markdown cell, there are typos "orginize" and "drectories".